Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/WindowsSceneUnderstandingObserver.cs

```diff
@@ -662,8 +665,17 @@ private async Task RunObserverAsync(CancellationToken cancellationToken)
                         }
                         await new WaitForUpdate();
 
-                        sceneToWorldTransformMatrix = GetSceneToWorldTransform();
-
+                        System.Numerics.Matrix4x4? transformResult = GetSceneToWorldTransform();
+                        if (transformResult.HasValue)
+                        {
+                            sceneToWorldTransformMatrix = transformResult.Value;
+                        }
+                        else
+                        {
+                            await new WaitForUpdate();
```

Comment:
To make sure I'm understanding this, this basically waits a frame and then retries to `GetSceneToWorldTransform`?

Reply:
Just discovered a miss thanks to this comment! So the behavior now is that it waits for a frame and then go to line 648 and then 654. By the way it seems that the first call to get the pose via OpenXR always fails for some reason.

Reply:
> By the way it seems that the first call to get the pose via OpenXR always fails for some reason.

Huh that seems unexpected :/ I guess good thing this works around it, but I'll file a bug on the plug-in too to investigate